flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/casting/CastRulesTest.java

```diff
@@ -136,6 +136,15 @@
                     FIELD("d", ARRAY(STRING())));
 
     Stream<CastTestSpecBuilder> testCases() {
+        return Stream.of(
+                CastTestSpecBuilder.testCastTo(CHAR(12))
+                        .fromCase(
+                                BINARY(4),
+                                new byte[] {-12, 32, 46, -72},
+                                fromString("f4202eb8    ")));
+    }
+
+    Stream<CastTestSpecBuilder> testCases2() {
```

Comment:
Why this?

Reply:
:( Very sorry, I forgot to revert, was testing to get the example code, fixed.